1.9/usage/pods/examples.md

```diff
@@ -4,8 +4,357 @@ feature_maturity: preview
+| environment                  | object  | Container environment variables. Can override pod environment variables. Must be capitalized.                           |
+| healthCheck                  |         |                                                                                                                         |
+| healthCheck.http                   |         | Protocol type. Options: `http`, `tcp`, `exec`.                                                                          |
+| healthCheck.http.endpoint           | string  | Endpoint name to use.                                                                                                   |
+| healthCheck.http.path               | string  | Path to the endpoint exposed by the task that provides health status.                                                   |
+| healthCheck.http.scheme             | string  | For httpHealthCheck, use `HTTP`.                                                                                        |
+| healthCheck.gracePeriodSeconds     | integer | Interval to ignore health check failures after a task is first started or until a task is first healthy (default: 300). |
+| healthCheck.intervalSeconds        | integer | Interval between health checks (default: 60).                                                                           |
+| healthCheck.maxConsecutiveFailures | integer | Number of consecutive failures before task is killed (default: 3).                                                      |
+| healthCheck.timeoutSeconds         | integer | Time to wait until health check is complete (default: 20).                                                              |
+| healthCheck.delaySeconds           | integer | Time to wait until starting health check (default: 2).                                                                  |
+| artifacts                    | array   | Array of artifact objects                                                                                               |
+| healthCheck.uri                    | strings | URI to resources to download (i.e., .tgz, tar.gz, .zip, .txz, etc).                                                     |
+| healthCheck.extract                | boolean | Extract fetched artifact.                                                                                               |
+| healthCheck.executable             | boolean | Set fetched artifact as executable.                                                                                     |
+| healthCheck.cache                  | boolean | Cache fetched artifact.                                                                                                 |
+| healthCheck.destPath               | strings | Destination path of artifact.                                                                                           |
+
+# Further examples
```

Comment:
They're not advanced, they're smaller than the complete pod. Not sure how to be more precise than `More examples`